AutomatedTesting/Gem/PythonTests/scripting/FileMenu_Default_NewAndOpen.py

```diff
@@ -50,38 +51,43 @@ class TestFileMenuDefaultNewOpen:
 
     @pyside_utils.wrap_async
     async def run_test(self):
-        # 1) Open Script Canvas window (Tools > Script Canvas)
+
+        # Preconditions
+        general.idle_enable(True)
+
+        # 1) Open Script Canvas window and wait for it to render(Tools > Script Canvas)
         general.open_pane("Script Canvas")
+        helper.wait_for_condition(lambda: general.is_pane_visible("Script Canvas"), TIME_TO_WAIT)
```

Comment:
Since this is used more than once or twice, you should store this into a variable so you don't get syntax errors and it's easier to update references in the event of a window title change.

Reply:
Now using const "SCRIPT_CANVAS_PANE"